### ISSUE TITLE
posts/2020/07/17/latex-on-ipad/

### ISSUE THREAD

**Issue by WeixuanZ** [WeixuanZ/utterances#7]
# Compiling LaTeX on an iPad · Weixuan

How you can actually compile a LaTeX document on an iPad, off-line and free.

[https://weixuanz.github.io/posts/2020/07/17/latex-on-ipad/](https://weixuanz.github.io/posts/2020/07/17/latex-on-ipad/)

**Comment by eleanor-clifford** [comment]
Just use Linux like a **real** programmer

**Comment by WeixuanZ** [comment]
https://insights.stackoverflow.com/survey/2020#technology-developers-primary-operating-systems

Seems that macOS > Linux 😉

**Comment by eleanor-clifford** [comment]
Right, just like a Ford Fiesta > Tesla Model S just because it has more sales...
